### ISSUE TITLE
Q. Which loss is used for validation? 

### ISSUE THREAD

**Issue by soumya997** [ayoolaolafenwa/PixelLib#75]
As per the [docs](https://pixellib.readthedocs.io/en/latest/custom_train.html), 

> mask_rcnn_model_25–0.55678, it is saved with its epoch number and its corresponding validation loss.

I just wanted to know which loss function is used for calculating the validation loss.
It would be very helpful if you @ayoolaolafenwa can answer that. 

Thank you.
